cpp/include/raft/core/span.hpp

```diff
@@ -120,22 +115,22 @@ class span {
 
   constexpr auto cend() const noexcept -> const_iterator { return data() + size(); }
 
-  _RAFT_HOST_DEVICE constexpr auto rbegin() const noexcept -> reverse_iterator
+  _CCCL_HOST_DEVICE constexpr auto rbegin() const noexcept -> reverse_iterator
```

Comment:
I prefer not to change this macro in this PR. The previous macro is used all over the place so any changes should be targeted in a separate PR to maintain cohesiveness.